lib/vrl/stdlib/src/is_ipv6.rs

```diff
@@ -0,0 +1,106 @@
+use std::net::Ipv6Addr;
+
+use ::value::Value;
+use vrl::prelude::*;
+
+fn is_ipv6(value: Value) -> Resolved {
+    let value_str = value.try_bytes_utf8_lossy()?;
+    Ok(value_str.parse::<Ipv6Addr>().is_ok().into())
+}
+
+#[derive(Clone, Copy, Debug)]
+pub struct IsIpv6;
+
+impl Function for IsIpv6 {
+    fn identifier(&self) -> &'static str {
+        "is_ipv6"
+    }
+
+    fn parameters(&self) -> &'static [Parameter] {
+        &[Parameter {
+            keyword: "value",
+            kind: kind::BYTES,
+            required: true,
+        }]
+    }
+
+    fn examples(&self) -> &'static [Example] {
+        &[
+            Example {
```

Comment:
Do we want a mirror'd "random string" example here too?

Reply:
Hmm, actually... I think I want to remove the "random string" one from the examples.

Thinking it over, it feels weird to show an example of how _not_ to use the function, since we generally don't see to do that in the existing VRL docs, and the unit tests are covering that case already.

I also don't actually know what these examples are used for. 🤷🏻

Reply:
I think the aim was eventually to drive the cue docs from these, but we never quite got there. I'd also be ok with dropping them.

Reply:
Hmmmm, if there's ever a future in using them to actually drive the docs... then I think I'll mirror over the "random string" example and leave them as-is.

Reply:
Fixed in 1a6099b52.